src/app/pages/home/education/education.scss

```diff
@@ -131,6 +195,13 @@
             text-transform: none;
             width: 27.76vw;
 
+            @include collapsed {
```

Comment:
`height: 7.88vw` on the `.btn` needs to go away. The element gets chopped on mobile 
![screenshot 2016-02-24 15 44 48](https://cloud.githubusercontent.com/assets/12257565/13302076/a03ee476-db0d-11e5-985a-3d0ec62a2dd7.png)